### PR TITLE
Add Washington DC to Master branch

### PR DESCRIPTION
**Pull request by calvinmichel**

### CODE REVIEW

File: places/calvinmichel-washingtondc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Washington, DC
+### Fun Facts
+### Museums
```
Comment (bjswick33) [inline places/calvinmichel-washingtondc]: Can't wait to visit!

**Comment by vicuk** [comment]
What's your favorite museum in DC? Your list is empty and I'm looking for a recommendation.

File: places/calvinmichel-washingtondc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## Washington, DC
```
Comment (realslimshanky) [inline places/calvinmichel-washingtondc]: Nice place

File: places/calvinmichel-washingtondc [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+### Fun Facts
+-  It's fun to put your feet in the Reflection Pool
+
+### Museums
+- Spy Museum
+- Air and Space Museum
```
Comment (sjhiltonjr) [inline places/calvinmichel-washingtondc]: Very cool!

Comment (sathya-ghub) [inline places/calvinmichel-washingtondc]: Udvar Hazy or the one in downtown DC?

**Pull request merged**